universal-login-contracts/test/contracts/ERC1077.js

```diff
@@ -16,29 +15,6 @@ const to = '0x0000000000000000000000000000000000000001';
 const {gasPrice} = DEFAULT_PAYMENT_OPTIONS;
 const overrideOptions = {gasPrice, gasLimit: 200000};
 
-describe('Void ERC1077', () => {
-  let provider;
-  let identityWithZeroKey;
-  let privateKey;
-  let signature;
-  let msg;
-  let wallet;
-
-  beforeEach(async () => {
-    provider = createMockProvider();
-    [, , , , , , , , , wallet] = getWallets(provider);
-    identityWithZeroKey = await deployContract(wallet, ERC1077, [constants.AddressZero]);
-  });
-
-  it('execute signed fails', async () => {
-    signature = [];
-    msg = {...transferMessage, from: identityWithZeroKey.address};
-    privateKey = wallet.privateKey;
-    await expect(identityWithZeroKey.executeSigned(...getExecutionArgs(msg), signature, overrideOptions))
-      .to.be.revertedWith('Invalid signature');
-  });
-});
```

Comment:
This needs to stay. This test makes sure we can create MasterCopy contract (an empty version of the contract, that serves as a library) and execution with it will always fail.

Reply:
We should probably make it more explicit. Perhaps using better description:
`Void ERC1077 (serves as master copy)`

Reply:
ops